### PR TITLE
fix: website check targets

### PR DESCRIPTION
**Pull request by afbjorklund**
### What does this PR do?

It was only possible to check the much bigger top-level targets...

But it should be possible to format and lint also the website dir.

### Screenshot/screencast of this PR

```console
$ cd website
$ yarn lint:check && yarn format:check
yarn run v1.22.19
$ eslint . --ext js,ts,tsx
=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=3.3.1 <5.1.0

YOUR TYPESCRIPT VERSION: 5.1.3

Please only submit bug reports when using the officially supported version.

=============
Done in 4.71s.
yarn run v1.22.19
$ prettier --check '**/*.{md,js}' 'src/**/*.{css,ts,tsx}'
Checking formatting...
All matched files use Prettier code style!
Done in 2.67s.

```
### How to test this PR?

`cd website`
`yarn run`

### CODE REVIEW

File: website/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (benoitf) [inline website/.prettierrc]: we shouldn't duplicate the top folder configuration that applies to all the files

Comment (benoitf) [reply website/.prettierrc]: By default prettier will also search in parent folder for a configuration

File: website/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{
```
Comment (benoitf) [inline website/.eslintrc.json]: we shouldn't redeclare a configuration.
The goal of the monorepository is to share the linter, formatter configuration across all the modules, not to fork/duplicate

**Review by benoitf — CHANGES_REQUESTED** [review]

File: website/.prettierrc (hunk 1 above)
Comment (afbjorklund) [reply website/.prettierrc:1]: there was some issue were something was looking for `website/website`

File: website/.eslintrc.json (hunk 2 above)
Comment (afbjorklund) [reply website/.eslintrc.json]: there was an issue with it:
```
Parsing error: Cannot read file '/home/anders/podman-desktop/website/website/tsconfig.json'
```

File: website/.prettierrc (hunk 1 above)
Comment (benoitf) [reply website/.prettierrc]: I think it's more for eslint twebsite/website) but we should still not duplicate linter/formatter rules across the projet

File: website/.eslintrc.json (hunk 2 above)
Comment (afbjorklund) [reply website/.eslintrc.json]: Not sure why it needed an extra period ?

```json
    "project": [
      "packages/*/tsconfig.json",
      "./website/tsconfig.json",
      "./extensions/*/tsconfig.json",
      "./tests/tsconfig.json",
      "./tools/tsconfig.json"
    ]

```

Comment (afbjorklund) [reply website/.eslintrc.json]: Did a workaround instead, apparently the feature is missing from json (only available in js)

`"tsconfigRootDir": __dirname`

File: website/package.json [hunk 3]
```diff
@@ -13,10 +13,12 @@
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
     "typecheck": "tsc",
-    "format:check": "prettier --check 'src/**/*.{css,ts,tsx}'",
-    "format:fix": "prettier --write 'src/**/*.{css,ts,tsx}'",
-    "lint:check": "eslint . --ext js,ts,tsx",
-    "lint:fix": "eslint . --fix --ext js,ts,tsx"
+    "format:check": "prettier --check '**/*.{md,js}' 'src/**/*.{css,ts,tsx}'",
```
Comment (benoitf) [inline website/package.json]:
```suggestion
    "format:check": "prettier --cache --check '**/*.{md,js}' 'src/**/*.{css,ts,tsx}'",
```

File: website/package.json [hunk 4]
```diff
@@ -13,10 +13,12 @@
...
+    "format:fix": "prettier --write '**/*.{md,js}' 'src/**/*.{css,ts,tsx}'",
```
Comment (benoitf) [inline website/package.json]:
```suggestion
    "format:fix": "prettier --cache --write '**/*.{md,js}' 'src/**/*.{css,ts,tsx}'",
```

File: website/package.json [hunk 5]
```diff
@@ -13,10 +13,12 @@
...
+    "markdownlint:check": "markdownlint-cli2 --config ../.markdownlint.yaml \"**/*.md\" \"#node_modules\"",
+    "markdownlint:fix": "markdownlint-cli2-fix --config ../.markdownlint.yaml \"**/*.md\" \"#node_modules\"",
+    "lint:check": "cd .. && eslint website --ext js,ts,tsx",
+    "lint:fix": "cd .. && eslint website --fix --ext js,ts,tsx"
```
Comment (benoitf) [inline website/package.json]: it's missing the cache option of parent folder (I don't know for the memory as it was reported by you and I'm not facing the issue)
https://github.com/containers/podman-desktop/blob/main/package.json#L65-L66

Comment (afbjorklund) [reply website/package.json]: then again this smaller check target doesn't run out of memory

Comment (benoitf) [reply website/package.json]: ok for ignoring env for memory but it should use the cache

**Review by benoitf — CHANGES_REQUESTED** [review]
just missing the cache option and we're good to merge after that

File: website/package.json (hunk 5 above)
Comment (afbjorklund) [reply website/package.json]: added cache, and a missing markdownlint target (which is failing)

**Comment by afbjorklund** [comment]
```console
$ yarn format:check && yarn lint:check
yarn run v1.22.19
$ prettier --cache --check '**/*.{md,js}' 'src/**/*.{css,ts,tsx}'
Checking formatting...
All matched files use Prettier code style!
Done in 2.84s.
yarn run v1.22.19
$ cd .. && eslint --cache --cache-location website/.eslintcache website --ext js,ts,tsx
Done in 5.16s.
$ yarn format:check && yarn lint:check
yarn run v1.22.19
$ prettier --cache --check '**/*.{md,js}' 'src/**/*.{css,ts,tsx}'
Checking formatting...
All matched files use Prettier code style!
Done in 0.60s.
yarn run v1.22.19
$ cd .. && eslint --cache --cache-location website/.eslintcache website --ext js,ts,tsx
Done in 1.42s.
```

**Review by benoitf — APPROVED** [review]
awesome, thanks @afbjorklund

**Comment by benoitf** [comment]
tested and worked as expected, merging

**Pull request merged**